endpoints.go

```diff
@@ -144,9 +176,11 @@ var (
 	EndpointEmoji         = func(eID string) string { return EndpointCDN + "emojis/" + eID + ".png" }
 	EndpointEmojiAnimated = func(eID string) string { return EndpointCDN + "emojis/" + eID + ".gif" }
 
-	EndpointOauth2            = EndpointAPI + "oauth2/"
```

Comment:
I agree that the name `EndpointOAuth2` fits better, but it seems like too minuscule a change to break existing code. I think defining `EndpointOAuth2` and then setting `EndpointOauth2 = EndpointOAuth2` to maintain backwards compatibility would be a good solution.